src/style.css

```diff
@@ -0,0 +1,126 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap');
+
+body {
+  width: 100%;
+  padding: 0;
+  margin: 0;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.navbar {
+  background-color: black;
+  color: white;
+}
+
+.logo {
+  width: 2rem;
+  height: 2rem;
+  margin: 1.5rem 1rem;
+  border-radius: 50%;
+}
+
+.navlinks {
+  list-style: none;
+  justify-content: space-around;
+  flex: 11;
+  padding-top: 14px;
+  padding-right: 0;
+  padding-inline-start: 0;
+}
+
+.navlinks a {
+  text-decoration: none;
+  color: white;
+}
+
+.navitems {
+  padding: 0 0.2rem;
+}
+
+.card-div {
+  background-color: bisque;
+  padding: 1rem;
+  margin: 2%;
+}
+
+.card-div:nth-child(odd) {
+  background-color: rgb(223, 142, 44);
+  padding: 1rem;
+  margin: 2%;
+}
+
+.meal-desc {
+  flex-direction: column;
+  text-align: right;
+}
+
+.btn-div {
+  flex-direction: column;
+  margin: 1rem 2rem 0 2rem;
+}
+
+.name-like {
+  justify-content: space-between;
+}
+
+.like-span {
+  align-self: center;
+}
+
+.likes-count {
+  color: black;
+  font-size: 1.3rem;
+  margin-right: 4px;
+}
+
+.reserve-btn,
+.comment-btn {
+  padding: 0.4rem;
+  margin: 0.4rem 0;
+  text-transform: uppercase;
+  background-color: brown;
+  color: white;
+  cursor: pointer;
+  border-radius: 8px;
+}
+
+.reserve-btn:hover,
+.comment-btn:hover {
+  background-color: white;
+  color: brown;
+}
+
+.meal-photo {
+  width: 100%;
+  justify-content: center;
+}
+
+.footer {
+  text-align: center;
+  padding: 1rem 0;
+  color: white;
+  background-color: black;
+}
+
+.footer a {
+  text-decoration: none;
+  color: white;
+}
+
+@media only screen and (min-width: 426px) {
+  .navbar {
+    padding: 0 15%;
+  }
+
+  .mainCont {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 30px;
+    grid-auto-rows: minmax(100px, auto);
+    background-color: #edeff5;
+    padding: 20px 15%;
+  }
```

Comment:
As requested by the previous reviewer kindly fix the issue of some contents and images falling off and overflowing the to the outside of the main content. As shown below you can see that the third cards in the grid columns are outside the `main content` container. Kindy fix this to make your project more responsive and up to standard. :wink:

Reply:
@jerryowusu I cannot understand what you are requesting.

![image](https://user-images.githubusercontent.com/53879944/219856803-fad2b22f-35cf-4844-8833-8f30bc2c13cf.png)

The above is the card and they appear correctly from my end.

Reply:
Kindly check the dev branch and ensure you have my latest changes.

Reply:
@DannieMo, yes I understand your concerns. However, since you are building a responsive website, all screen widths of the project should look identical to this, and I can say the desktop version looks perfectly fine just as you have posted above but the mobile version is where the problem is. Kindly check the images of the mobile screens below: 


![Screenshot from 2023-02-18 10-42-02](https://user-images.githubusercontent.com/65233860/219864866-56f2b986-bfdc-479c-8242-914db3f866d1.png)


![Screenshot from 2023-02-18 10-42-36](https://user-images.githubusercontent.com/65233860/219864875-a54e24fd-b9ba-4e31-bc2f-d66cbc4d5986.png)

Reply:
@jerryowusu the screen size with the width 501 is not a real device width, I tested my code against all mobile screens (320-428px), tablet screen (768px) and desktops of all size. So, I don't understand why you've used 501 and 565 to test the code.